crypto/decrepit/cfb/cfb.c

```diff
@@ -36,6 +40,55 @@ static int aes_cfb_init_key(EVP_CIPHER_CTX *ctx, const uint8_t *key,
   return 1;
 }
 
+static int aes_cfb1_cipher(EVP_CIPHER_CTX *ctx, uint8_t *out,
+                            const uint8_t *in, size_t len) {
+  if (!out || !in) {
+    return 0;
+  }
+
+  EVP_CFB_CTX *cfb_ctx = ctx->cipher_data;
+  if (ctx->flags & EVP_CIPH_FLAG_LENGTH_BITS) {
```

Comment:
There is no way to set this flag for a consumer.

Reply:
Yes, I know this: aws-lc doesn't define the cipher with this flag and does not have any EVP flag setter API. These words maybe also applicable to ossl. I did not find related EVP setter API. I don't understand original purpose(for development like stitch code?). I added this in 2nd commit just to avoid surprise(e.g. the flag can be turned)

Reply:
There is a setter function in OSSL: https://github.com/openssl/openssl/blob/master/crypto/evp/evp_lib.c#L1108

> But I added this in 2nd commit just to avoid surprise

Okay, that make sense I guess.